### DISCUSSION TITLE
Pimcore X and Mysql 8

### DISCUSSION THREAD

**Discussion by johanlindgren** [pimcore/pimcore#9302] [General]
The requirement  Mysql >= 8.0 limits the use of ASW Aurora, Is this a hard requirement or could you look into to having only optional functionality depending on Mysql 8 support?

**Comment by jdreesen** [comment]
This is the (only?) feature requiring MySQL >= 8.0 afaik: #7503

As it handles the inheritance of `DataObject`s I don't think it is optional...

**Comment by nirbanc2020** [comment]
I am using one custom bundle that was used in Pimcore 6.8 and the same I am using in Pimcore X.But why I am geting the lock symbol in bundle list page
![project-presales-local-Pimcore](https://user-images.githubusercontent.com/73533273/120625659-8ec38100-c47f-11eb-95c5-589385810ce1.png)
?

**Reply by jdreesen** [reply]
That's a completely different issue. Please continue in https://github.com/pimcore/pimcore/discussions/9321

**Comment by IgorBenko** [comment]
So this means that there is currently no way to run Pimcore X on Aurora Serverless, right?

This limit is pretty hard. On the other side there is also no support for redis clusters before Pimcore X (AFAIK). So we either stay on 6.9 without redis cluster support or we upgrade and we can't use Aurora Serverless.

Does anyone have any suggestion what would be a good alternative then? 

I also see that Aurora AWS completely disappeared from the list of supported Databases 🙁

**Comment by johanlindgren** [comment]
Any comments from the core team? @brusch ?

Will there be any support for Aurora Serverless setup in PImcore? We need to understand which effort we should put into finding alternatives here

Cheers and thanks for any feedback

**Reply by fashxp** [reply]
We where hoping, that aurora will support Mysql8 soon ... see also https://forums.aws.amazon.com/thread.jspa?threadID=306735 
In the time being, you have to use RDS.

The change that requires MySQL 8 (see #7503) is a quite central one...

**Reply by johanlindgren** [reply]
Thanks for your quick reply

**Reply by IgorBenko** [reply]
@fashxp @johanlindgren Aurora is now compatible with MySQL 8.0. Unfortunately not yet with Aurora Serverless, but hopefully that comes soon 🤞 https://aws.amazon.com/de/blogs/database/amazon-aurora-mysql-3-with-mysql-8-0-compatibility-is-now-generally-available/

**Reply by fashxp** [reply]
step by step ;-)
